clients/thymioupgrader/ThymioUpgrader.cpp

```diff
@@ -58,7 +58,7 @@ namespace Aseba
 	{
 		// Create the gui ...
 		setWindowTitle(tr("Thymio Firmware Upgrader"));
-		
+        setWindowIcon(QIcon(":/images/thymioupgrader.svgz"));
```

Comment:
it was correct in QT creator an bad on github... is there a way to control that?